compatibility.bs

```diff
@@ -502,6 +502,12 @@ The following <code>-webkit-</code> <a>vendor prefixed</a> properties must be su
 <br>
 <div class="note">As soon as each property is unprefixable it can be defined as a "simple" alias.</div>
 
+<h4 id="css-non-alias">Non-alias vendor prefixed properties</h4>
```

Comment:
I think "non-aliased" reads as more natural, at least to me. But I could be convinced either way.

(unless non-alias is the more commonly used technical term i'm not familiar with)

Reply:
(feel free to change and merge after -- or just merge if you think `non-alias` is better here)